### DISCUSSION TITLE
When do you use the "dev" scripts in db / api etc

### DISCUSSION THREAD

**Discussion by ck-euan** [t3-oss/create-t3-turbo#1002] [Q&A] [answered]
Hi guys,
I am having TypeScript performance issues in my app which I think is due to my large DB schema.
Whenever I have the dist/ folder performance is obviously much better

I might be misunderstanding something with this setup, but what should the process be for creating the dist/ folders?
I can see that the api / db packages now have dev scripts, but when do we run them?

E.g. do I run `pnpm dev -F db` whenever I do new DB changes? 
This seems about as annoying as Prisma generate.

Is it possible to make running something like `pnpm dev -F nextjs` also run the dev scripts for api / db?
I tried making dev depend on all other "^dev" scripts, but can't because of the `persistent: true` in turbo.json

Any advice or recommendations would be appreciated :)

**Comment by juliusmarminge** [comment, marked as answer]
With the new turbo ui, i usually run pnpm dev in root and that runs everything.

The syntax for starting scripts in dependent workspaces is `turbo dev --filter nextjs...` (where ... is the thing that tells turbo to run the pipeline for nextjs and its dependencies)

**Reply by ck-euan** [reply]
Ahh amazing thank you! The `...` was the thing I was missing! 😅
